idiss/Cargo.toml

```diff
@@ -8,13 +9,14 @@ license-file = "../../LICENSE-APACHE"
 description = "Exports of the identity issuance functionality needed by the identity provider. Currently supports use from nodejs."
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
+[profile.release.build-override]
+opt-level = 3
```

Comment:
Can you please add the same comment here that we have in the mobile-wallet/Cargo.toml as to why this is necessary